bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/ImageRenderer.java

```diff
@@ -32,6 +32,8 @@
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
```

Comment:
No need for these imports if nothing is logged by this class.

Reply:
Removed, thank you!